documentation/Performance-Tweaks-Guide.md

```diff
@@ -0,0 +1,156 @@
+## Performance Tweaks Guid
+
+During the latest (start of June 2024) round of performance improvements we had to do quite a lot of performance related test, which were best done on the actual demo server.
+In that time I accumulated quite a few CLI commands that can be helpful in the future.
+
+## Getting inside the docker container
+
+After ssh to the demo server.
+
+```bash
+docker container ls
+```
+
+If container name is not indicative of the url it is exposed in, then you can find more information via, `cat /etc/nginx/sites-enabled/default` for NGINX or `~/demo_server/env_files` for Caddy
```

Comment:
The env files were there for nginx too, so its slightly confusing to mention them separately. I would just say that the port that each container is mapped to is defined in the `env_files` and not mention the server apps here.

Reply:
Cool, i just removed the mention of nginx or caddy entirely since the mapping should be in the name: [Remove mention of caddy or nginx](https://github.com/msupply-foundation/conforma-server/pull/1110/commits/e893655e4a16da091d35b38533d8a33863db9d8c)